### PR TITLE
test/k8s: remove istio.go test

### LINKED ISSUE #24475: CI: `K8sAgentIstioTest Istio Bookinfo Demo Tests bookinfo inter-service connectivity` failing with `Istio sidecar proxies are not reachable` on Kernel 4.19 / K8s 1.17 - 1.24
### Test Name

```shell
K8sAgentIstioTest Istio Bookinfo Demo Tests bookinfo inter-service connectivity
```

### Failure Output

```shell
Istio sidecar proxies are not reachable
```

### Stack Trace

```shell
/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:515
Timed out after 245.544s.
Istio sidecar proxies are not reachable
Expected
    <bool>: false
to be true
/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/istio.go:425
```

### Standard Output

```shell
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 0
Number of "level=warning" in logs: 0
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
No errors/warnings found in logs
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 0
Number of "level=warning" in logs: 0
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
No errors/warnings found in logs
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 0
Number of "level=warning" in logs: 5
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
Top 3 errors/warnings:
Key allocation attempt failed
Policy map sync fixed errors, consider running with debug verbose = policy to get detailed dumps
UpdateIdentities: Skipping Delete of a non-existing identity
Cilium pods: [cilium-25ft8 cilium-8jbfg]
Netpols loaded: 
CiliumNetworkPolicies loaded: default::cnp-specs 
Endpoint Policy Enforcement:
Pod                                    Ingress   Egress
reviews-v2-77946fdc74-ddp82            false     false
istio-ingressgateway-78fd9db7c-78slb   false     false
istiod-84cb48bbc5-ccqqh                false     false
coredns-6b4fc58d47-7jjc8               false     false
details-v1-fcf5ddfbb-ncdgs             false     false
productpage-v1-56cbdf9897-k897f        false     false
ratings-v1-d9b7b74b-8q27z              false     false
reviews-v1-59dccf6f9b-z9btg            false     false
Cilium agent 'cilium-25ft8': Status: Ok  Health: Ok Nodes "" ContainerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 33 Failed 0
Cilium agent 'cilium-8jbfg': Status: Ok  Health: Ok Nodes "" ContainerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 41 Failed 0
```

### Standard Error

```shell
14:24:46 STEP: Running BeforeAll block for EntireTestsuite K8sAgentIstioTest
14:24:46 STEP: Ensuring the namespace kube-system exists
14:24:47 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=cilium-test-logs")
14:24:47 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=cilium-test-logs") => <nil>
14:24:47 STEP: Downloading cilium-istioctl
14:24:47 STEP: Installing Cilium
14:24:48 STEP: Waiting for Cilium to become ready
14:25:19 STEP: Validating if Kubernetes DNS is deployed
14:25:19 STEP: Checking if deployment is ready
14:25:19 STEP: Checking if kube-dns service is plumbed correctly
14:25:19 STEP: Checking if DNS can resolve
14:25:19 STEP: Checking if pods have identity
14:25:22 STEP: Kubernetes DNS is up and operational
14:25:22 STEP: Validating Cilium Installation
14:25:22 STEP: Performing Cilium controllers preflight check
14:25:22 STEP: Performing Cilium health check
14:25:22 STEP: Performing Cilium status preflight check
14:25:22 STEP: Checking whether host EP regenerated
14:25:29 STEP: Performing Cilium service preflight check
14:25:29 STEP: Performing K8s service preflight check
14:25:36 STEP: Waiting for cilium-operator to be ready
14:25:36 STEP: WaitforPods(namespace="kube-system", filter="-l name=cilium-operator")
14:25:36 STEP: WaitforPods(namespace="kube-system", filter="-l name=cilium-operator") => <nil>
14:25:36 STEP: Labeling default namespace for sidecar injection
14:25:36 STEP: Setting label istio-injection=enabled in namespace default
14:25:36 STEP: Deploying Istio
14:26:06 STEP: Waiting for Istio pods to be ready
14:26:06 STEP: WaitforNPodsRunning(namespace="istio-system", filter="-l istio")
14:26:06 STEP: WaitforNPods(namespace="istio-system", filter="-l istio") => <nil>
14:26:06 STEP: WaitforPods(namespace="istio-system", filter="-l istio")
14:26:06 STEP: WaitforPods(namespace="istio-system", filter="-l istio") => <nil>
14:26:06 STEP: Waiting for Istio service "istio-ingressgateway" to be ready
14:26:06 STEP: Waiting for Istio service "istiod" to be ready
14:26:06 STEP: Waiting for DNS to resolve Istio service "istio-ingressgateway"
14:26:06 STEP: Waiting for DNS to resolve Istio service "istiod"
14:26:07 STEP: Creating policy in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/cnp-specs.yaml"
14:26:13 STEP: Creating resources in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/bookinfo-v2.yaml"
14:26:13 STEP: Creating resources in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/bookinfo-v1.yaml"
14:26:14 STEP: Waiting for Bookinfo pods to be ready
14:26:14 STEP: WaitforPods(namespace="default", filter="-l zgroup=bookinfo")
14:26:46 STEP: WaitforPods(namespace="default", filter="-l zgroup=bookinfo") => <nil>
14:26:46 STEP: Waiting for Bookinfo endpoints to be ready
14:26:48 STEP: Waiting for Bookinfo service "details" to be ready
14:26:48 STEP: Waiting for Bookinfo service "ratings" to be ready
14:26:48 STEP: Waiting for Bookinfo service "reviews" to be ready
14:26:49 STEP: Waiting for Bookinfo service "productpage" to be ready
14:26:49 STEP: Waiting for DNS to resolve Bookinfo service "productpage"
14:26:49 STEP: Waiting for DNS to resolve Bookinfo service "reviews"
14:26:49 STEP: Waiting for DNS to resolve Bookinfo service "ratings"
14:26:49 STEP: Waiting for DNS to resolve Bookinfo service "details"
14:26:49 STEP: Testing Istio service configuration
14:26:50 STEP: Checking that Istio proxy config at "reviews-v1-59dccf6f9b-z9btg" has service "ratings.default.svc.cluster.local" on port "9080" for "outbound"
14:26:50 STEP: Checking that Istio proxy config at "ratings-v1-d9b7b74b-8q27z" has service "" on port "9080" for "inbound"
14:26:50 STEP: Checking that Istio proxy config at "productpage-v1-56cbdf9897-k897f" has service "details.default.svc.cluster.local" on port "9080" for "outbound"
14:26:50 STEP: Checking that Istio proxy config at "details-v1-fcf5ddfbb-ncdgs" has service "" on port "9080" for "inbound"
14:26:50 STEP: Checking that Istio proxy config at "productpage-v1-56cbdf9897-k897f" has service "ratings.default.svc.cluster.local" on port "9080" for "outbound"
14:26:50 STEP: Testing service local access
14:26:50 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" can connect to "http://127.0.0.1:9080/health"
14:26:50 STEP: Checking that "ratings-v1-d9b7b74b-8q27z" can connect to "http://127.0.0.1:9080/health"
14:26:51 STEP: Checking that "ratings-v1-d9b7b74b-8q27z" can connect to "http://127.0.0.1:9080/ratings/0"
14:26:51 STEP: Checking that "ratings-v1-d9b7b74b-8q27z" can connect to "http://127.0.0.1:9080/health"
14:26:51 STEP: Checking that "details-v1-fcf5ddfbb-ncdgs" can connect to "http://127.0.0.1:9080/health"
14:26:51 STEP: Checking that "productpage-v1-56cbdf9897-k897f" can connect to "http://127.0.0.1:9080/health"
14:26:51 STEP: Testing L7 filtering
14:26:51 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" can connect to "ratings.default.svc.cluster.local:9080/health"
14:26:51 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
Was able to connect from "reviews-v1-59dccf6f9b-z9btg" to "ratings.default.svc.cluster.local:9080/ratings/0", but expected no connection: Exitcode: 0 
Stdout:
 	 
Stderr:
 	 --2023-03-20 14:26:51--  http://ratings.default.svc.cluster.local:9080/ratings/0
	 Resolving ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)... 10.101.216.139
	 Connecting to ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)|10.101.216.139|:9080... connected.
	 HTTP request sent, awaiting response... 200 OK
	 Length: unspecified [application/json]
	 Saving to: '0'
	 
	      0K                                                        7.48M=0s
	 
	 2023-03-20 14:26:51 (7.48 MB/s) - '0' saved [48]
	 
	 

14:26:51 STEP: Checking that "productpage-v1-56cbdf9897-k897f" can connect to "details.default.svc.cluster.local:9080/health"
14:26:51 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/health"
14:27:22 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
14:27:53 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" can connect to "ratings.default.svc.cluster.local:9080/health"
14:27:53 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
Was able to connect from "reviews-v1-59dccf6f9b-z9btg" to "ratings.default.svc.cluster.local:9080/ratings/0", but expected no connection: Exitcode: 0 
Stdout:
 	 
Stderr:
 	 --2023-03-20 14:27:53--  http://ratings.default.svc.cluster.local:9080/ratings/0
	 Resolving ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)... 10.101.216.139
	 Connecting to ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)|10.101.216.139|:9080... connected.
	 HTTP request sent, awaiting response... 200 OK
	 Length: unspecified [application/json]
	 Saving to: '0.1'
	 
	      0K                                                        6.96M=0s
	 
	 2023-03-20 14:27:53 (6.96 MB/s) - '0.1' saved [48]
	 
	 

14:27:53 STEP: Checking that "productpage-v1-56cbdf9897-k897f" can connect to "details.default.svc.cluster.local:9080/health"
14:27:53 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/health"
14:28:23 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
14:28:54 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" can connect to "ratings.default.svc.cluster.local:9080/health"
14:28:55 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
Was able to connect from "reviews-v1-59dccf6f9b-z9btg" to "ratings.default.svc.cluster.local:9080/ratings/0", but expected no connection: Exitcode: 0 
Stdout:
 	 
Stderr:
 	 --2023-03-20 14:28:55--  http://ratings.default.svc.cluster.local:9080/ratings/0
	 Resolving ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)... 10.101.216.139
	 Connecting to ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)|10.101.216.139|:9080... connected.
	 HTTP request sent, awaiting response... 200 OK
	 Length: unspecified [application/json]
	 Saving to: '0.2'
	 
	      0K                                                        7.62M=0s
	 
	 2023-03-20 14:28:55 (7.62 MB/s) - '0.2' saved [48]
	 
	 

14:28:55 STEP: Checking that "productpage-v1-56cbdf9897-k897f" can connect to "details.default.svc.cluster.local:9080/health"
14:28:55 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/health"
14:29:25 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
14:29:56 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" can connect to "ratings.default.svc.cluster.local:9080/health"
14:29:56 STEP: Checking that "reviews-v1-59dccf6f9b-z9btg" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
Was able to connect from "reviews-v1-59dccf6f9b-z9btg" to "ratings.default.svc.cluster.local:9080/ratings/0", but expected no connection: Exitcode: 0 
Stdout:
 	 
Stderr:
 	 --2023-03-20 14:29:56--  http://ratings.default.svc.cluster.local:9080/ratings/0
	 Resolving ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)... 10.101.216.139
	 Connecting to ratings.default.svc.cluster.local (ratings.default.svc.cluster.local)|10.101.216.139|:9080... connected.
	 HTTP request sent, awaiting response... 200 OK
	 Length: unspecified [application/json]
	 Saving to: '0.3'
	 
	      0K                                                        7.84M=0s
	 
	 2023-03-20 14:29:56 (7.84 MB/s) - '0.3' saved [48]
	 
	 

14:29:56 STEP: Checking that "productpage-v1-56cbdf9897-k897f" can connect to "details.default.svc.cluster.local:9080/health"
14:29:56 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/health"
14:30:27 STEP: Checking that "productpage-v1-56cbdf9897-k897f" cannot connect to "ratings.default.svc.cluster.local:9080/ratings/0"
FAIL: Timed out after 245.544s.
Istio sidecar proxies are not reachable
Expected
    <bool>: false
to be true
=== Test Finished at 2023-03-20T14:30:57Z====
14:30:57 STEP: Running JustAfterEach block for EntireTestsuite K8sAgentIstioTest
===================== TEST FAILED =====================
14:30:57 STEP: Running AfterFailed block for EntireTestsuite K8sAgentIstioTest
cmd: kubectl get pods -o wide --all-namespaces
Exitcode: 0 
Stdout:
 	 NAMESPACE           NAME                                   READY   STATUS    RESTARTS   AGE     IP              NODE   NOMINATED NODE   READINESS GATES
	 cilium-monitoring   grafana-585bb89877-xcb26               0/1     Running   0          46m     10.0.0.219      k8s2   <none>           <none>
	 cilium-monitoring   prometheus-8885c5888-w76zp             1/1     Running   0          46m     10.0.0.180      k8s2   <none>           <none>
	 default             details-v1-fcf5ddfbb-ncdgs             2/2     Running   0          4m48s   10.0.1.173      k8s1   <none>           <none>
	 default             productpage-v1-56cbdf9897-k897f        2/2     Running   0          4m48s   10.0.1.96       k8s1   <none>           <none>
	 default             ratings-v1-d9b7b74b-8q27z              2/2     Running   0          4m49s   10.0.1.8        k8s1   <none>           <none>
	 default             reviews-v1-59dccf6f9b-z9btg            2/2     Running   0          4m48s   10.0.0.206      k8s2   <none>           <none>
	 default             reviews-v2-77946fdc74-ddp82            2/2     Running   0          4m49s   10.0.0.7        k8s2   <none>           <none>
	 istio-system        istio-ingressgateway-78fd9db7c-78slb   1/1     Running   0          5m12s   10.0.0.127      k8s2   <none>           <none>
	 istio-system        istiod-84cb48bbc5-ccqqh                1/1     Running   0          5m25s   10.0.0.111      k8s2   <none>           <none>
	 kube-system         cilium-25ft8                           1/1     Running   0          6m14s   192.168.56.11   k8s1   <none>           <none>
	 kube-system         cilium-8jbfg                           1/1     Running   0          6m14s   192.168.56.12   k8s2   <none>           <none>
	 kube-system         cilium-operator-6476d88777-4m2ql       1/1     Running   0          6m14s   192.168.56.12   k8s2   <none>           <none>
	 kube-system         cilium-operator-6476d88777-tfr4t       1/1     Running   0          6m14s   192.168.56.11   k8s1   <none>           <none>
	 kube-system         coredns-6b4fc58d47-7jjc8               1/1     Running   0          6m16s   10.0.0.222      k8s2   <none>           <none>
	 kube-system         etcd-k8s1                              1/1     Running   0          51m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-apiserver-k8s1                    1/1     Running   0          51m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-controller-manager-k8s1           1/1     Running   0          51m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-proxy-gvbrw                       1/1     Running   0          51m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-proxy-lt2mk                       1/1     Running   0          47m     192.168.56.12   k8s2   <none>           <none>
	 kube-system         kube-scheduler-k8s1                    1/1     Running   0          51m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         log-gatherer-bjmhs                     1/1     Running   0          46m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         log-gatherer-gkspn                     1/1     Running   0          46m     192.168.56.12   k8s2   <none>           <none>
	 kube-system         registry-adder-96rd2                   1/1     Running   0          47m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         registry-adder-dzpd4                   1/1     Running   0          47m     192.168.56.12   k8s2   <none>           <none>
	 
Stderr:
 	 

Fetching command output from pods [cilium-25ft8 cilium-8jbfg]
cmd: kubectl exec -n kube-system cilium-25ft8 -c cilium-agent -- cilium endpoint list
Exitcode: 0 
Stdout:
 	 ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                                  IPv6        IPv4         STATUS   
	            ENFORCEMENT        ENFORCEMENT                                                                                                        
	 124        Enabled            Disabled          2546       k8s:app=details                                              fd02::1e1   10.0.1.173   ready   
	                                                            k8s:io.cilium.k8s.namespace.labels.istio-injection=enabled                                    
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                      
	                                                            k8s:io.cilium.k8s.policy.istiosidecarproxy=true                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                               
	                                                            k8s:io.kubernetes.pod.namespace=default                                                       
	                                                            k8s:istio.io/rev=default                                                                      
	                                                            k8s:security.istio.io/tlsMode=istio                                                           
	                                                            k8s:service.istio.io/canonical-name=details                                                   
	                                                            k8s:service.istio.io/canonical-revision=v1                                                    
	                                                            k8s:track=stable                                                                              
	                                                            k8s:version=v1                                                                                
	                                                            k8s:zgroup=bookinfo                                                                           
	 268        Disabled           Disabled          1          k8s:cilium.io/ci-node=k8s1                                                            ready   
	                                                            k8s:node-role.kubernetes.io/master                                                            
	                                                            reserved:host                                                                                 
	 751        Enabled            Disabled          63135      k8s:app=ratings                                              fd02::1f4   10.0.1.8     ready   
	                                                            k8s:io.cilium.k8s.namespace.labels.istio-injection=enabled                                    
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                      
	                                                            k8s:io.cilium.k8s.policy.istiosidecarproxy=true                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                               
	                                                            k8s:io.kubernetes.pod.namespace=default                                                       
	                                                            k8s:istio.io/rev=default                                                                      
	                                                            k8s:security.istio.io/tlsMode=istio                                                           
	                                                            k8s:service.istio.io/canonical-name=ratings                                                   
	                                                            k8s:service.istio.io/canonical-revision=v1                                                    
	                                                            k8s:version=v1                                                                                
	                                                            k8s:zgroup=bookinfo                                                                           
	 1619       Disabled           Disabled          58033      k8s:app=productpage                                          fd02::10d   10.0.1.96    ready   
	                                                            k8s:io.cilium.k8s.namespace.labels.istio-injection=enabled                                    
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                      
	                                                            k8s:io.cilium.k8s.policy.istiosidecarproxy=true                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                               
	                                                            k8s:io.kubernetes.pod.namespace=default                                                       
	                                                            k8s:istio.io/rev=default                                                                      
	                                                            k8s:security.istio.io/tlsMode=istio                                                           
	                                                            k8s:service.istio.io/canonical-name=productpage                                               
	                                                            k8s:service.istio.io/canonical-revision=v1                                                    
	                                                            k8s:track=stable                                                                              
	                                                            k8s:version=v1                                                                                
	                                                            k8s:zgroup=bookinfo                                                                           
	 3761       Disabled           Disabled          4          reserved:health                                              fd02::1af   10.0.1.242   ready   
	 
Stderr:
 	 

cmd: kubectl exec -n kube-system cilium-8jbfg -c cilium-agent -- cilium endpoint list
Exitcode: 0 
Stdout:
 	 ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                                                    IPv6       IPv4         STATUS   
	            ENFORCEMENT        ENFORCEMENT                                                                                                                         
	 121        Disabled           Disabled          4          reserved:health                                                                fd02::27   10.0.0.90    ready   
	 516        Disabled           Disabled          22511      k8s:app=istiod                                                                 fd02::50   10.0.0.111   ready   
	                                                            k8s:install.operator.istio.io/owning-resource=unknown                                                          
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                                       
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=istiod-service-account                                                 
	                                                            k8s:io.kubernetes.pod.namespace=istio-system                                                                   
	                                                            k8s:istio.io/rev=default                                                                                       
	                                                            k8s:istio=pilot                                                                                                
	                                                            k8s:operator.istio.io/component=Pilot                                                                          
	                                                            k8s:sidecar.istio.io/inject=false                                                                              
	 714        Disabled           Disabled          58584      k8s:app=istio-ingressgateway                                                   fd02::93   10.0.0.127   ready   
	                                                            k8s:chart=gateways                                                                                             
	                                                            k8s:heritage=Tiller                                                                                            
	                                                            k8s:install.operator.istio.io/owning-resource=unknown                                                          
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                                       
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=istio-ingressgateway-service-account                                   
	                                                            k8s:io.kubernetes.pod.namespace=istio-system                                                                   
	                                                            k8s:istio.io/rev=default                                                                                       
	                                                            k8s:istio=ingressgateway                                                                                       
	                                                            k8s:operator.istio.io/component=IngressGateways                                                                
	                                                            k8s:release=istio                                                                                              
	                                                            k8s:service.istio.io/canonical-name=istio-ingressgateway                                                       
	                                                            k8s:service.istio.io/canonical-revision=latest                                                                 
	                                                            k8s:sidecar.istio.io/inject=false                                                                              
	 967        Disabled           Disabled          3923       k8s:app=reviews                                                                fd02::98   10.0.0.206   ready   
	                                                            k8s:io.cilium.k8s.namespace.labels.istio-injection=enabled                                                     
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                                       
	                                                            k8s:io.cilium.k8s.policy.istiosidecarproxy=true                                                                
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                                                
	                                                            k8s:io.kubernetes.pod.namespace=default                                                                        
	                                                            k8s:istio.io/rev=default                                                                                       
	                                                            k8s:security.istio.io/tlsMode=istio                                                                            
	                                                            k8s:service.istio.io/canonical-name=reviews                                                                    
	                                                            k8s:service.istio.io/canonical-revision=v1                                                                     
	                                                            k8s:track=stable                                                                                               
	                                                            k8s:version=v1                                                                                                 
	                                                            k8s:zgroup=bookinfo                                                                                            
	 1823       Disabled           Disabled          36913      k8s:io.cilium.k8s.policy.cluster=default                                       fd02::ea   10.0.0.222   ready   
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=coredns                                                                
	                                                            k8s:io.kubernetes.pod.namespace=kube-system                                                                    
	                                                            k8s:k8s-app=kube-dns                                                                                           
	 3427       Disabled           Disabled          4919       k8s:app=reviews                                                                fd02::18   10.0.0.7     ready   
	                                                            k8s:io.cilium.k8s.namespace.labels.istio-injection=enabled                                                     
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                                                       
	                                                            k8s:io.cilium.k8s.policy.istiosidecarproxy=true                                                                
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                                                
	                                                            k8s:io.kubernetes.pod.namespace=default                                                                        
	                                                            k8s:istio.io/rev=default                                                                                       
	                                                            k8s:security.istio.io/tlsMode=istio                                                                            
	                                                            k8s:service.istio.io/canonical-name=reviews                                                                    
	                                                            k8s:service.istio.io/canonical-revision=v2                                                                     
	                                                            k8s:version=v2                                                                                                 
	                                                            k8s:zgroup=bookinfo                                                                                            
	 3845       Disabled           Disabled          1          k8s:cilium.io/ci-node=k8s2                                                                             ready   
	                                                            reserved:host                                                                                                  
	 
Stderr:
 	 

===================== Exiting AfterFailed =====================
14:31:10 STEP: Running AfterEach for block EntireTestsuite K8sAgentIstioTest Istio Bookinfo Demo
14:31:10 STEP: Deleting resource in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/bookinfo-v2.yaml"
14:31:10 STEP: Deleting resource in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/bookinfo-v1.yaml"
14:31:10 STEP: Deleting policy in file "/home/jenkins/workspace/cilium-master-k8s-1.17-kernel-4.19/src/github.com/cilium/cilium/test/k8s/manifests/cnp-specs.yaml"
14:31:11 STEP: Running AfterEach for block EntireTestsuite

[[ATTACHMENT|693362fc_K8sAgentIstioTest_Istio_Bookinfo_Demo_Tests_bookinfo_inter-service_connectivity.zip]]
14:31:12 STEP: Running AfterAll block for EntireTestsuite K8sAgentIstioTest
14:31:12 STEP: Deleting default namespace sidecar injection label
14:31:12 STEP: Setting label istio-injection- in namespace default
14:31:12 STEP: Deleting the Istio resources
14:31:13 STEP: Waiting all terminating PODs to disappear
14:31:27 STEP: Deleting the istio-system namespace
14:31:27 STEP: Deleting namespace istio-system
14:31:37 STEP: Removing Cilium installation using generated helm manifest
```

### Resources

- Jenkins URL: https://jenkins.cilium.io/job/cilium-master-k8s-1.17-kernel-4.19/lastCompletedBuild/testReport/Suite-k8s-1/17/K8sAgentIstioTest_Istio_Bookinfo_Demo_Tests_bookinfo_inter_service_connectivity/
- ZIP file(s): [693362fc_K8sAgentIstioTest_Istio_Bookinfo_Demo_Tests_bookinfo_inter-service_connectivity.zip](https://github.com/cilium/cilium/files/11021344/693362fc_K8sAgentIstioTest_Istio_Bookinfo_Demo_Tests_bookinfo_inter-service_connectivity.zip)

### Anything else?

Consistently failing since March 10 on Kernel 4.19 pipelines for K8s 1.17 - 1.24.

### PR DESCRIPTION
**Pull request by aanm**
This test has been failing quarantined. Since we are not planning on fixing it because it will require too much effort, we should remove it from the CI.

Fixes https://github.com/cilium/cilium/issues/24475

### CODE REVIEW

File: test/k8s/istio.go [hunk 1]
```diff
@@ -1,430 +0,0 @@
...
-			dnsChecks := []string{productPage, reviews, ratings, details}
-			app := "app"
-			health := "health"
-			ratingsPath := "ratings/0"
-			apiPort := "9080"
-			podNameFilter := "{.items[*].metadata.name}"
-
-			bookinfoV1YAML := helpers.ManifestGet(kubectl.BasePath(), "bookinfo-v1.yaml")
-			bookinfoV2YAML := helpers.ManifestGet(kubectl.BasePath(), "bookinfo-v2.yaml")
-			l7PolicyPath := helpers.ManifestGet(kubectl.BasePath(), "cnp-specs.yaml")
```
Comment (tklauser) [inline test/k8s/istio.go:319]: I think we can also remove these manifest files from `test/k8s/manifests` as part of this PR.

**Review by christarazi — APPROVED** [review]
Pending Tobias's suggestion

**Review by tklauser — APPROVED** [review]

**Pull request merged**